app/grandchallenge/challenges/models.py

```diff
@@ -906,6 +911,22 @@ class ChallengeRequestStatusChoices(models.TextChoices):
             MaxValueValidator(limit_value=60),
```

Comment:
@amickan @jmsmkn Are `blank=True` and `null=True` correct for this field? On the form we set the related field to required, so I'm a bit confused why blank is allowed here.

Reply:
I do not think that is correct. I think that is the same for all of these fields:

- inference_time_limit_in_minutes
- average_size_of_test_image_in_mb
- phase_1_number_of_submissions_per_team
- phase_2_number_of_submissions_per_team
- phase_1_number_of_test_images
- phase_2_number_of_test_images
- budget_for_hosting_challenge
- long_term_commitment (boolean, so unsure)
- data_license (boolean, so unsure)
- algorithm_inputs
- algorithm_outputs
- challenge_fee_agreement (boolean, so unsure)

Maybe Anne can confirm?

Reply:
They should indeed all be required. Back when we first implemented this form, users could choose between Type 1 and Type 2 challenges and for the former the budget fields were not required. When we removed the type choice, looks like we didn't migrate the schema, but only made the fields required always on the form.

Reply:
Thanks for the clarification! Will update this in this PR.

Reply:
The phase 2 number of submissions and number of test images need to allow 0s (see this example: https://grand-challenge.org/documentation/create-your-own-challenge/#example-of-a-challenge-with-more-than-2-phases)

Reply:
Okay, just learned that the PositiveIntegerField allows zero by default. So nothing needed there.